bimsb/packages/staging.scm

```diff
@@ -2665,3 +2665,32 @@ to estimate cell type proportions in bulk RNA-seq data.")
 algorithm} (DSA) for extracting cell-type specific gene expression profiles
 from mixed tissue samples.")
       (license license:gpl2))))
+
+(define-public r-sanssouci
+  (let ((commit "5fe20a9aaf4ac637fa83d9cc73ff1c22de97ca6f")
+        (revision "1"))
+    (package
+      (name "r-sanssouci")
+      (version (string-append "0-" revision "." (string-take commit 9)))
```

Comment:
You can use `git-version` in these cases, e.g. `(version (git-version "0" revision commit))`

Reply:
Thank you. Fixed.